kubemarine/plugins/calico.py

```diff
@@ -118,8 +120,9 @@ def enrich_configmap_calico_config(cluster, obj_list):
     key = "ConfigMap_calico-config"
     val = cluster.inventory['plugins']['calico']['mtu']
     obj_list[key]['data']['veth_mtu'] = str(val)
-    cluster.log.verbose(f"The {key} has been patched in 'data.typha_service_name' with '{val}'")
-    if cluster.inventory['plugins']['calico']['typha']['enabled'] == True:
+    cluster.log.verbose(f"The {key} has been patched in 'data.veth_mtu' with '{val}'")
+    cluster.log.verbose(f"ENABLE: {cluster.inventory['plugins']['calico']['typha']['enabled']}")
+    if cluster.inventory['plugins']['calico']['typha']['enabled'] == 'true':
```

Comment:
If I correctly understand, there can be both `enabled: true` and `enabled: "true"`. The first is python boolean, the second is python "true" string. To handle both cases, [is_true](https://github.com/Netcracker/KubeMarine/blob/main/kubemarine/jinja.py#L31) filter was developed.

Reply:
Since it is in defaults.yaml, it is a public api and can be redefined by user to `enabled: something`